scripts/get

```diff
@@ -62,7 +63,7 @@ runAsRoot() {
 # verifySupported checks that the os/arch combination is supported for
 # binary builds.
 verifySupported() {
-  local supported="darwin-amd64\nlinux-386\nlinux-amd64\nlinux-arm\nlinux-arm64\nlinux-ppc64le\nwindows-amd64"
+  local supported="darwin-amd64\nlinux-386\nlinux-amd64\nlinux-arm\nlinux-arm64\nlinux-ppc64le\nlinux-mips64le\nwindows-amd64"
```

Comment:
this can be removed. This PR does not provide a mips64le architecture build for Helm 2.

Reply:
Ok, let me modify it

Reply:
I had remove.@bacongobbler